### DISCUSSION TITLE
How does Web3Modal V2 listen to the event accountsChanged

### DISCUSSION THREAD

**Discussion by rick20220222** [WalletConnect/walletconnect-monorepo#1726] [Developer Support]
How does Web3Modal V2 listen to the event accountsChanged
![screenshot-20221207-144347](https://user-images.githubusercontent.com/100116396/206107682-aab1448a-0635-444c-9f8d-b1d6f8e303c5.png)

**Comment by xzilja** [comment]
This is more in the realm of [wagmi](https://wagmi.sh/) you can grab provider from there and add this listener.

**Reply by devon-bfs** [reply]
How do you get the accounts normally. On connect?
